.gitignore

```diff
@@ -41,3 +41,4 @@ dump.rdb
 /devenv.nix
 /flake.lock
 /flake.nix
+/qodana.yaml
```

Comment:
?

Reply:
It's a tool to analyze code. I can remove it from here but IMO it's not a big deal to add it in gitignore.